src/models/real-fevr-opener.ts

```diff
@@ -168,8 +166,28 @@ export class OpenerRealFvr extends Model<OpenerRealFvrMethods> implements Deploy
     return this.callTx(this.contract.methods.getRegisteredIDs(_address));
   }
 
+  async getMarketplaceDistributionForERC721(tokenId: number) {
+    const res = await this.callTx(this.contract.methods.getMarketplaceDistributionForERC721(tokenId));
+
+    return {
```

Comment:
provide an util that makes this logic so we can have this typed when needed and provide the user a easy way for them to create data with this same format. (check `src/utils/`) for examples

Reply:
done